neos/neos-ui
# TASK: Refactor Validator translations to Neos.Neos.Ui package

**What I did**

I added the translations of our validators from the Neos.Neos package directly too the Neos.Neos.Ui package.

**How I did it**

Moved the translations over, and adjusted the Validator Components to load the correct translations.

**How to verify it**

Checkout this PR in combination with: https://github.com/neos/neos-development-collection/pull/4493

and add a Validator to your NodeType. You should see, that the translation is still working.
